mlflow/tracking/fluent.py

```diff
@@ -1296,6 +1300,26 @@ def print_auto_logged_info(r):
                   'training_mse': 1.9721522630525295e-31}
         tags: {'estimator_class': 'sklearn.linear_model._base.LinearRegression',
                'estimator_name': 'LinearRegression'}
+
+    Note that framework-specific configurations set at any point will
+    take precedence over any configurations set by this function. For example:
+
+    .. code-block:: python
+
+        mlflow.autolog(log_models=False, exclusive=True)
+        import sklearn
+
+    would enable autologging for `sklearn` with `log_models=False` and `exclusive=True`,
+    but
+
+    .. code-block:: python
+
+        mlflow.autolog(log_models=False, exclusive=True)
+        import sklearn
+        mlflow.sklearn.autolog(log_models=True)
+
+    would enable autologging for `sklearn` with `log_models=True` and `exclusive=False`,
+    the latter resulting from the default value for `exclusive` in `mlflow.sklearn.autolog`.
```

Comment:
We should move this up above the params and code snippet  (after line 1234).

Reply:
Done!